lib/rules/jsx-pascal-case.js

```diff
@@ -60,6 +60,20 @@ function testAllCaps(name) {
   return true;
 }
 
+function testValidity(name, allowAllCaps, ignore) {
+  const isPascalCase = testPascalCase(name);
+  const isAllowedAllCaps = allowAllCaps && testAllCaps(name);
+  const isIgnored = ignore.some(
+    (entry) => name === entry || minimatch(name, entry, {noglobstar: true})
+  );
+
+  if (!isPascalCase && !isAllowedAllCaps && !isIgnored) {
+    const message = allowAllCaps ? 'usePascalOrSnakeCase' : 'usePascalCase';
+
+    throw new Error(message);
```

Comment:
i'm not a fan of using errors for flow control here. let's find a more explicit mechanism.

Reply:
Hope fully my news changes are a bit more elegant than using error handling to implement the rule